platform/lib/pipeline/growReferenceChecker.js

```diff
@@ -79,9 +79,16 @@ class GrowReferenceChecker {
       stream.pipe(gulp.dest('./'));
 
       stream.on('end', () => {
-        this._log.complete(`Finished fixing. A total of ${this._brokenReferencesCount} had ` +
-        `errors. ${this._unfindableDocuments.length + Object.keys(this._multipleMatches).length}` +
-        'still have.');
+        if (Object.keys(this._multipleMatches).length == 0
+            && this._unfindableDocuments.length == 0) {
+          this._log.success('All references intact!');
+          process.exit(0);
```

Comment:
I think it's better to let the caller decide how to handle the success case. Otherwise it's impossible to call this from a gulp task.